src/sso.ts

```diff
@@ -15,6 +15,10 @@ export default class MetricsSSO extends Metrics {
     return response.valid
   }
 
+  async token (params: { clientIdentifier: string, clientSecret: string, authorizationCode?: string, refreshToken?: string, grantType: oAuthGrantTypes }) {
```

Comment:
This name needs to be more explicit. `token` is generic and doesn't explain what this method is used for.